docs/cluster-deployment/stripe.rst

```diff
@@ -0,0 +1,164 @@
+using your credit card. Here we will explain how to do so, from the initial
+subscribing to the deployment itself. Our payment processing and billing are
+supported by `Stripe`_.
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+
+.. _signup-stripe:
+
+Subscribe to CrateDB Cloud
+==========================
+
+When you have completed the initial signup process, you will see the CrateDB
+Cloud Console organization overview. This normally lists all your subscriptions
+and the clusters that can be deployed in them. In this case, however, you
+probably do not yet have any. You will therefore only see a set of subscription
+options, as well as a popup warning that you need a subscription to deploy
+services on CrateDB Cloud:
+
+.. image:: ../_assets/img/stripe-subscribe.png
+   :alt: CrateDB Cloud Console overview
+
+As you can see, although CrateDB Cloud supports subscribing via the Marketplace
+offers, there is a more straightforward way. Simply click the blue button
+marked *Create new subscription*.
+
+
+.. _signup-stripe-plans:
+
+Choose a plan
+=============
+
+You will now see an overview of the different subscription plans CrateDB Cloud
+offers. For details on what each plan entails, see our `reference on
+subscription plans`_.
+Choose a plan to your liking by clicking *Choose*.
+
+.. image:: ../_assets/img/stripe-plans.png
+   :alt: CrateDB Cloud subscription plans
+
+
+.. _signup-stripe-configure:
+
+Configure your cluster
+======================
+
+Now comes the stage to finalize the subscription and configure your soon-to-be-
+deployed cluster. The next part will take you through the CrateDB Cloud
+cluster configuration wizard. In four quick steps, you will have everything
+set up.
+
+
+Wizard step 1
+-------------
+
+In the first step of the wizard, you are prompted for an organization, a
+project name, and a project region. Fortunately, the organization you have
+created earlier is already pre-selected for you. The region is also set by
+default, but can be changed using the region dropdown menu. All that is needed
+is to name the project within which your cluster will be deployed.
+
+.. image:: ../_assets/img/stripe-wizard-step1.png
+   :alt: CrateDB Cloud configuration wizard step 1
+
+Click *Next* to proceed.
+
+
+Wizard step 2
+-------------
+
+In the next step, we come to the cluster itself. Here the wizard will ask you
+to name the cluster, as well as the username and the password that will
+subsequently be used to access the cluster via the unique cluster URL. The
+password must be at least 24 characters long; any characters are accepted,
+including special characters. If you want, click the *Auto-generate password*
+button to automatically generate a secure 24 character password (if it shows
+a password already, clicking again will generate a new one).
+
+.. WARNING::
+
+    The password will not be retrievable past this point. You may want to copy
```

Comment:
This is going to be not true anymore with the next release of the UI, should we preempt it? re: that modal popup that Lewis showed.

Reply:
Ok given the UI release, I will change this now